### DISCUSSION TITLE
cite scikit-build in the academic paper

### DISCUSSION THREAD

**Discussion by njzjz** [scikit-build/scikit-build#914] [Q&A] [answered]
Hi,

I am writing an academic paper to describe a software package that uses `scikit-build` to build the Python package. I mention scikit-build in the paper, and I'd like to know whether there is a recommended way to cite `scikit-build`. Thanks!

**Comment by njzjz** [comment]
I found a conference paper here https://conference.scipy.org/proceedings/scipy2022/henry_schreiner.html , which looks good to me.

**Reply by jcfr** [reply]
Thanks for asking.

We will follow up shortly. The link you provided is related to scikit-hep which is different.

**Reply by jcfr** [reply, marked as answer]
Here is a pull-request:
* https://github.com/scikit-build/scikit-build/pull/916

**Reply by njzjz** [reply]
Thanks!
